### ISSUE TITLE
Why does this require activesupport? 

### ISSUE THREAD

**Issue by Leonas** [patleb/sunzistrano#1]
I'd like to test this out on some nodejs and non-rails ruby apps but will the rails dependency prevent this?

**Comment by patleb** [comment]
Ooops, sorry for the delay, I'm not in the watchers of my own repo :S

The dependency is for the tool, not for the projects configured. Basically, some stuff like `with_indifferent_access` are used, so it's a hard dependency on ActiveSupport.

You could take a look at an example [here](https://github.com/patleb/sunzistrano-gitlab), which was a project configured when Sunzistrano was in version 1.0. The Readme hasn't been updated yet, but now it's a bit different in version 2.0. Most common recipes/roles/files for a full fledge Rails project are now in this repo and used by default, but can be overriden in your project by creating a file with the same relative path (ex.: `config/sunzi/roles/deployer.sh`).

So, basically, to answer your question: no, it won't prevent you to use it for non-rails or ruby projects.
